app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col-md-4">
+    <h1>Top Movies</h1>
+      <% count = 0 %>
```

Comment:
This and the code below should be at the same indentation level as `<h1>` since it's not included in the header :)